content/docs/2.16/setupscaler.md

```diff
@@ -12,7 +12,7 @@ description = "Procedure to Setup a Scaler in KEDA"
 
 2. **KEDA Installation**:
    - KEDA needs to be installed on your Kubernetes cluster before you can use it.
-   - Follow the [KEDA installation guide](https://keda.sh/docs/2.14/deploy/) carefully, including any prerequisites specific to your Kubernetes setup.
+   - Follow the [KEDA installation guide](https://keda.sh/docs/deploy/) carefully, including any prerequisites specific to your Kubernetes setup.
```

Comment:
nit: I think it might be even better to use relative path
```suggestion
   - Follow the [KEDA installation guide](../deploy/) carefully, including any prerequisites specific to your Kubernetes setup.
```

Reply:
I fixed this in all versions down to 2.14.